tools/spectral/ipa/rulesets/functions/IPA102CollectionIdentifierCamelCase.js

```diff
@@ -22,10 +22,12 @@ export default (input, options, { path, documentInventory }) => {
   const oas = documentInventory.resolved;
   const pathKey = input;
 
-  // Check for exception at the path level
-  if (hasException(oas.paths[input], RULE_NAME)) {
-    collectException(oas.paths[input], RULE_NAME, path);
-    return;
+  if (oas.paths[input]) {
+    const pathItem = oas.paths[input];
+    if (hasExceptionInEveryHttpMethod(pathItem, RULE_NAME)) {
+      collectException(pathItem, RULE_NAME, path);
```

Comment:
Collect exception needs message but we have multiple exceptions here. 
Leaving as draft to discuss approach.